### PR TITLE
Wash position:fixed style in HTML mail for better security (#5264)

### PR DESCRIPTION
**Pull request by alecpl**
Possible solution for #5264 Though via PR as I don't really like the idea of replacing position fixed with absolute.

### CODE REVIEW

**Comment by alecpl** [comment]
@thomascube please review.

File: program/lib/Roundcube/rcube_washtml.php [hunk 1]
```diff
@@ -727,10 +732,9 @@ public static function fix_broken_lists(&$html)
      */
     protected function explode_style($style)
     {
-        $style = trim($style);
+        $pos = 0;
```
Comment (thomascube) [inline program/lib/Roundcube/rcube_washtml.php]: What is this change supposed to fix or improve in this matter?

Comment (alecpl) [inline program/lib/Roundcube/rcube_washtml.php]: There's no need to trim before we remove comments, better to trim after removing comments.

Comment (thomascube) [inline program/lib/Roundcube/rcube_washtml.php]: OK. This seemed like an irrelevant change to the actual topic of this PR

**Comment by thomascube** [comment]
I'm good with this change. position:fixed wouldn't fit well into the embedded message preview thus replacing it with absolute can make sense regardless the XSS situation.

**Pull request merged**